src/test/java/com/hyunec/cosmicbaseballinit/acceptancetest/NormalBaseballLv1Test.java

```diff
@@ -1,18 +1,55 @@
 package com.hyunec.cosmicbaseballinit.acceptancetest;
 
+import com.hyunec.cosmicbaseballinit.model.BattingResult;
+import com.hyunec.cosmicbaseballinit.service.BattingServiceImpl;
+import org.assertj.core.data.Percentage;
 import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.RepeatedTest;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
 
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.hyunec.cosmicbaseballinit.model.BattingResult.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
 class NormalBaseballLv1Test {
+
+    @Autowired
+    private BattingServiceImpl battingServiceImpl;
+
     @DisplayName("타격 결과는 모두 같은 확률을 가집니다.")
-    @Test
+    @RepeatedTest(100)
     void t1() {
-        throw new RuntimeException("Not yet implemented");
+
+        double caseSize = 100000;
+        double count = 0;
+
+        BattingResult randomResult = battingServiceImpl.batting();
+
+        for (double i = 0; i < caseSize; i++) {
+            BattingResult result = battingServiceImpl.batting();
+            if (randomResult == result) {
+                count++;
+            }
+        }
+
+        assertThat(count).isCloseTo(33333, Percentage.withPercentage(5));
```

Comment:
```suggestion
        assertThat(count).isCloseTo(caseSize/3, Percentage.withPercentage(5));
```

`nit` 
보통 매직 넘버는 사용을 지양하는 것을 추천합니다.
하지만 예외적으로 테스트 코드에서는 어느정도 재사용성을 포기하거나, 매직 넘버를 써도 좋다고 생각합니다.